docs/components/menus.md

```diff
@@ -0,0 +1,353 @@
+
+Dropdown menus display a list of options, triggered by an icon, button, or action. Their placement varies based on the element that opens them.
+
+#### Dropdown menu example
+
+The following is an example of a dropdown menu:
+
+- [Class definition](https://api.flutter.dev/flutter/material/PopupMenuButton-class.html)
+- [GitHub source](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/popup_menu.dart)
+- [Dartpad Demo](https://dartpad.dev/embed-flutter.html?gh_owner=material-components&gh_repo=material-components-flutter&gh_path=docs/components/dartpad/menus/dropdown&gh_ref=develop)
+
+![Dropdown menu example](assets/menus/dropdown_menu_example.png)
+
+```dart
+PopupMenuButton(
+  icon: Icon(Icons.more_vert),
+  itemBuilder: (BuildContext context) => <PopupMenuEntry>[
+    const PopupMenuItem(
+      child: ListTile(
+        leading: Icon(Icons.add),
+        title: Text('Item 1'),
+      ),
+    ),
+    const PopupMenuItem(
+      child: ListTile(
+        leading: Icon(Icons.anchor),
+        title: Text('Item 2'),
+      ),
+    ),
+    const PopupMenuItem(
+      child: ListTile(
+        leading: Icon(Icons.article),
+        title: Text('Item 3'),
+      ),
+    ),
+    const PopupMenuDivider(),
+    const PopupMenuItem(child: Text('Item A')),
+    const PopupMenuItem(child: Text('Item B')),
+  ],
+),
+```
+
+### Exposed dropdown menus
+
+Exposed dropdown menus display the currently selected menu item above the list of options. Some variations can accept user-entered input.
+
+#### Exposed dropdown menu example
+
+The following is an example of a dropdown menu:
+
+- [Class definition](https://api.flutter.dev/flutter/material/DropdownButton-class.html)
+- [GitHub source](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/dropdown.dart)
+- [Dartpad Demo](https://dartpad.dev/embed-flutter.html?gh_owner=material-components&gh_repo=material-components-flutter&gh_path=docs/components/dartpad/menus/exposed_dropdown&gh_ref=develop)
+
+![Exposed Dropdown menu example](assets/menus/exposed_dropdown_menu_example.png)
+
+```dart
+DropdownButton(
+  value: dropdownValue,
+  items: <DropdownMenuItem>[
+    DropdownMenuItem(
+      value: 'Option 1',
+      child: Text('Option 1'),
+    ),
+    DropdownMenuItem(
+      value: 'Option 2',
+      child: Text('Option 2'),
+    ),
+    DropdownMenuItem(
+      value: 'Option 3',
+      child: Text('Option 3'),
+    ),
+    DropdownMenuItem(
+      value: 'Option 4',
+      child: Text('Option 4'),
+    ),
+    DropdownMenuItem(
+      value: 'Option 5',
+      child: Text('Option 5'),
+    ),
+    DropdownMenuItem(
+      value: 'Option 6',
+      child: Text('Option 6'),
+    ),
+],
+  onChanged: (value) {
+    setState(() {
+      dropdownValue = value;
+    });
+  },
+),
+```
+
+## Theming
+
+Menus support [Material Theming](https://material.io/components/chips/#theming) and can be customized in terms of color, typography and shape.
```

Comment:
I think that material.io link is wrong, it refers to chips

Reply:
Done